gamebook/game_manager.py

```diff
@@ -0,0 +1,42 @@
+from gamebook.terminal_output import TerminalOutput
+from gamebook.terminal_input import TerminalInput
+import sys
+
+
+class GameManager(object):
+    def __init__(self, scene):
+        self.scene = scene
+        self.output_instance = TerminalOutput()
+        self.input_instance = TerminalInput()
+
+    def start(self, scene_import):
+        while True:
+            next_scene_name = self.run_scene()
```

Comment:
Write a comment that `run_scene` evaluates the scene and returns the next scene.